packages/ui-components/Common/Notification/index.module.css

```diff
@@ -13,8 +13,64 @@
     dark:bg-neutral-900;
 }
 
+.root[data-state='open'] {
+  animation: slide-in 300ms cubic-bezier(0.16, 1, 0.3, 1);
+}
+
+.root[data-state='closed'] {
+  animation: slide-out 200ms ease-in forwards;
+}
+
+.root[data-swipe='move'] {
+  transform: translateX(var(--radix-toast-swipe-move-x));
+}
+
+.root[data-swipe='cancel'] {
+  transform: translateX(0);
+  transition: transform 200ms ease-out;
+}
+
+.root[data-swipe='end'] {
+  animation: swipe-out 150ms ease-out forwards;
+}
```

Comment:
A) I'm not sure how I feel about the added complexity of a swipe
B) use tailwinds `@apply`